.github/workflows/release-please.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+      - main
```

Comment:
since the MPA repo requires a merge to `main` and then a merge to `prod`, what are thoughts on the branch flow? Should we have release PRs created when we merge to `prod`? or when we merge to `main`? I lean towards managing the release notes etc at the `main` branch level.